Makefile.toml

```diff
@@ -1292,11 +1292,12 @@ echo "All processes has exited."
 
 [tasks.slt]
 category = "RiseDev - SQLLogicTest"
-install_crate = { version = "0.17.0", crate_name = "sqllogictest-bin", binary = "sqllogictest", test_arg = [
+install_crate = { version = "0.17.1", crate_name = "sqllogictest-bin", binary = "sqllogictest", test_arg = [
   "--help",
 ], install_command = "binstall" }
 command = "sqllogictest"
 args = ["${@}"]
+env = { SLT_PORT = "4566", SLT_DB = "dev" }
```

Comment:
So that `risedev slt` doesn't need to add `-p 4566 -d dev` any more. (If flags are present, they should have higher precedence)

Reply:
What about leveraging the `risedev_env` way? 

https://github.com/risingwavelabs/risingwave/blob/2ab9b70b4d3c649b5bf51c216074b487db9e3016/src/risedevtool/src/risedev_env.rs

Reply:
Why?

Reply:
Hard-coded 4566 is not always the listen port. 🤔